src/System Application/App/Password/src/PasswordDialogManagement.Codeunit.al

```diff
@@ -116,7 +116,7 @@ codeunit 9810 "Password Dialog Management"
     /// Opens a dialog for the user to change a password and returns the old and new typed passwords if there is no validation error,
     /// otherwise an empty text are returned.
     /// </summary>
-    /// <param name="OldPassword">Out parameter, the old password user typed on the dialog.</param>
+    /// <param name="OldPassword">Out parameter, the old password to compare with the password user typed on the dialog.</param>
```

Comment:
We should specify OldPassword is also an input parameter. From this I expect the function itself determines the old password, compares and gives me the old password in case I want to do some kind of comparison.

Reply:
I agree now this is input and output parameter

Reply:
It probably makes no sense to return OldPassword at all, since we introduce checks and standards at the PasswordDialogManagment level.

Perhaps the best way would be to mark the old method as obsolete in future releases and create a new one (but would have to give a new name, number of parameters the same).

@AndreasMoth what do you think?

Reply:
Yes, that makes sense, to at least remove "var" from the OldPassword (through obsoletion, remember CLEAN tags).
You can also argue if we should even return the password, why does the caller need to get that password? But for now, just remove the "var" from OldPassword. We could also rename OldPassword to CurrentPassword while Password should be NewPassword (after all, you give the current password and get out the new password - renaming is not super important though).